examples/design/EU-DEMO/EUDEMO_reactor.py

```diff
@@ -237,6 +237,9 @@
             "nx": 2,
             "ny": 2,
         },
+        "PF Coils": {
+            "runmode": "read",
+        },
```

Comment:
Can we add plots for the PF coils to the example too? Or at least add them to the xz and xy plots.

Reply:
> Just a couple of bits. The resulting PF positioning doesn't seem quite right too, as they end up intersecting the outer leg of the TF coils (presumably something isn't being updated after the TF coil optimisation has run?).
> 
> ![image](https://user-images.githubusercontent.com/66615563/148815955-203bed9e-cfdd-401c-8dcd-2922daff2c5e.png)

So here we're just reading the PF coils from the reference eqdsk, which was made without any knowledge of what the TF coils should look like. The correct PF coil positions should be delivered with the "run" mode, which I will work on in a separate PR. It's unfortunate, but this state will just leave us with a non-sensical design.